packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -48,7 +48,11 @@ contract Bridge is EssentialContract, IBridge {
 
     /// Allow Bridge to receive ETH from the TokenVault or EtherVault.
     receive() external payable {
-        // TODO(dave,PR#13110): require the sender is the TokenVault or EtherVault
+        // Ensure the sender is either the Ether vault or the token vault.
```

Comment:
@d1onys1us  I believe this state works. If ether_vault is address 0, msg.sender == ether_vault will never be true.